src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java

```diff
@@ -88,9 +87,40 @@ public StochasticTestSuiteTest(String path) {
 
   }
 
-  @Ignore
   @Test
   public void testModel() throws IOException {
+
+    long[] stochasticSeeds = new long[]{
+            1595487468503L, 1595487468503L, 1595487468503L, 1595487468503L, 1595487468503L,
```

Comment:
If you use the same seed for everything just do the initialization with a one liner (Arrays.fill as above) instead of defining for every element.

Reply:
For this, I am using the same seed values for most of the test cases but test cases 28, 29, and 37 are having different seed values. But yes I am updating with Arrays.fill and updating the seed values of left 3 test cases separately.